packages/drivers/odsp-driver/src/odspDriverUrlResolver.ts

```diff
@@ -100,7 +100,7 @@ export class OdspDriverUrlResolver implements IUrlResolver {
             }
         }
 
-        const summarizer = request.headers?.[DriverHeader.summarizingClient];
+        const summarizer = !!request.headers?.[DriverHeader.summarizingClient];
```

Comment:
Doesn't this change `summarizer` changes from type `string | undefined` to `boolean`?

Reply:
yep. it is used later on with a definite `boolean` type.